Pull Requests/Backend-PRs.md

```diff
@@ -0,0 +1,30 @@
+# Backend Pull Requests
+
+This file details the practices that should be followed when creating a backend pull request (PR), in addition to the [general PR practices](README.md).
+
+## Table of Contents
+
+- [Backend Pull Requests](#backend-pull-requests)
+  - [Table of Contents](#table-of-contents)
+  - [Guidelines](#guidelines)
+    - [Added/Updated Routes](#addedupdated-routes)
+    - [Schema Definitions Changed](#schema-definitions-changed)
+    - [Updates to Wikis](#updates-to-wikis)
+
+## Guidelines
+
+Backend PRs should contain added/updated routes, schema definitions changed, and/or any updates to wikis required if the PR required changes to them.
```

Comment:
In order to reduce confusion, can we instead have something like "Backend description of PRs should contain..." or "Backend PRs should contain a written description of all added/updated routes, schema definitions changed,..."